tests/test_date.nit

```diff
@@ -0,0 +1,13 @@
+import date
```

Comment:
There is nothing tested in this file apart of the compilation. No asserts, no checks, just prints. How can you call that a test file?